test/jasmine-node/server.js

```diff
@@ -14,3 +17,36 @@ server.url = 'http://localhost:3000/genotet';
 
 /** @const {string} */
 server.uploadURL = 'http://localhost:3000/genotet/upload';
+
+/**
+ * @typedef {{
+ *   error: {type: string, message: string}
+ * }}
+ */
+server.uploadResponse;
```

Comment:
Oops looks like I made a mistake here. This should be uppercase.
```js
server.UploadResponse;
```
Please help me correct this and the 3 usages.